server/license.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	pluginapi "github.com/mattermost/mattermost-plugin-api"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const (
+	e20          = "E20"
+	professional = "professional"
+	enterprise   = "enterprise"
+)
+
+func HasEnterpriseFeatures(config *model.Config, license *model.License) bool {
+	if license != nil && (license.SkuShortName == e20 || license.SkuShortName == enterprise || license.SkuShortName == professional) {
```

Comment:
Just to double check: The plugin should be able to run with Professional?

Reply:
@hanzei Yes this is correct.

> https://pkg.go.dev/github.com/mattermost/mattermost-plugin-api#IsE20LicensedOrDevelopment got updated recently. Should this PR should make use of it?

That function can't be used for this reason, there is no check for Professional in that function.